contents/docs/[01]installation/[01]macos/index.md

```diff
@@ -58,11 +58,11 @@ mint run toucansites/toucan
 
 ## Prebuilt Binary from GitHub Releases
 
-You can also download Toucan directly from [GitHub Releases](https://github.com/toucansites/toucan/releases). This includes a prebuilt universal binaries for macOS.
+Toucan is also available for direct download from [GitHub Releases](https://github.com/toucansites/toucan/releases), which includes prebuilt universal binaries for macOS.
 
 ### Available Artifact from 1.0.0.beta.4
```

Comment:
Do we want to use an exact version here? `beta.4` could be an example after a generic description. Same below